### PR TITLE
SW-1863 Add search filters to inventory table

### PR DESCRIPTION
**Pull request by constanzauanini**
Screen Shot 2022-10-19 at 17 35 12

Screen Shot 2022-10-19 at 17 28 54

### CODE REVIEW

File: src/components/Inventory/index.tsx [hunk 1]
```diff
@@ -81,61 +85,93 @@ export default function Inventory(props: InventoryProps): JSX.Element {
...
-                  },
-                ],
-              },
-              {
-                operation: 'field',
-                field: 'organization_id',
-                values: [organization.id],
-              },
-            ],
+  const getParams = useCallback(() => {
```
Comment (karthikbtf) [inline src/components/Inventory/index.tsx:88]: wondering if all this is better pushed into the table component...
we would need some way to identify the empty state though.

**Review by karthikbtf — COMMENTED** [review]
@constanzauanini looks good,,, i think the filters need to be multiple selection.. with search looking for results in any of the selected nurseries.

**Review by karthikbtf — APPROVED** [review]
approving for now, follow up with @chudo76 whether this needs to be multiple-select filter?

File: src/components/Inventory/InventoryTable.tsx [hunk 2]
```diff
@@ -9,6 +9,9 @@ import { useDeviceInfo } from '@terraware/web-components/utils';
 import { APP_PATHS } from 'src/constants';
 import TextField from '@terraware/web-components/components/Textfield/Textfield';
 import InventoryCellRenderer from './TableCellRenderer';
```
Comment (karthikbtf) [inline src/components/Inventory/InventoryTable.tsx:11]: unrelated to change but might be good to rename this file to InventoryCellRenderer. The function name inside is InventoryCellRenderer.. and imported here as such as well.

File: src/components/Inventory/index.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/Inventory/index.tsx:88]: Yes, I tried to do that but I was repeating code to see if the inventory has columns or not

**Pull request merged**